crates/daphne/src/protocol/report_init.rs

```diff
@@ -67,11 +76,11 @@ impl InitializedReport<()> {
 }
 
 impl InitializedReport<WithPeerPrepShare> {
-    pub fn from_leader(
+    pub fn from_leader<'s>(
         decrypter: &impl HpkeDecrypter,
         valid_report_range: Range<messages::Time>,
         task_id: &TaskId,
-        task_config: &DapTaskConfig,
+        task_config: impl Into<PartialDapTaskConfig<'s>>,
```

Comment:
Can you remind me what "partial" here means?

Reply:
Just enough to initialize reports

Reply:
Why not use the entire `DapTaskConfig`?

Reply:
I will have to serialize it later and it's very big and cumbersome to serialize (manually because of capnproto)

Reply:
How about `PartialDapTaskConfigForReportInit`?

Reply:
Works for me!